js/pages/prediction/components/plp-inspector.js

```diff
@@ -1,4 +1,4 @@
-define(['knockout', 'jquery', 'text!./plp-inspector.html', 'appConfig', 'd3', 'css!./styles/plp.css'],
+define(['knockout', 'jquery', 'text!./plp-inspector.html', 'appConfig', 'd3', 'css!../../../styles/plp.css'],
```

Comment:
@anthonysena , since the `styles` directory is in the app root, let's use an absolute path (and, in general, three folders back doesn't looks to be very solid approach, easy to break accidentally)

Reply:
Agreed - made that change. Thanks!

Reply:
@anthonysena , thank you. I also see such thing in `js/pages/prediction/components/plp-roc.js`

Reply:
Missed that - think that should resolve this. Let me know if you have any other concerns.